GEOS_OceanGridComp.F90

```diff
@@ -9,6 +9,9 @@ module GEOS_OceanGridCompMod
 
   use ESMF
   use MAPL
+#ifdef BUILD_MIT_OCEAN
+  use MIT_GEOS5PlugMod, only: MITSetServices => SetServices  ! this sets IRF
```

Comment:
This is not a _clean_ way. We taken the effort **not** to have such _directions.
Maybe @heimbach can help get rid of this, in future?

Reply:
We know that this is needed to work with MITgcm, and I think none of us like it but all agreed to take it.